fixed-price-subscriptions/server/ruby/server.rb

```diff
@@ -83,13 +85,15 @@
   # Set the default payment method on the customer
   Stripe::Customer.update(
     data['customerId'],
-    invoice_settings: { default_payment_method: data['paymentMethodId'] }
+    invoice_settings: {
+      default_payment_method: data['paymentMethodId']
```

Comment:
This should probably be updated to match the use of a `payment_method` variable as here: https://github.com/stripe-samples/subscription-use-cases/pull/74/files#diff-2626038fecb4b14911cdcaf9422af8549a0a4a1adf02e965a1aa09a4296103f3R56
instead of pulling from the body data again.

Reply:
yeah I'm thinking of entirely removing retry-invoice as I don't think it actually works right now :(